arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/mmu/opcode/Create.java

```diff
@@ -45,6 +45,7 @@ public Create(final Hub hub) {
                 hub.currentFrame()
                     .frame()
                     .shadowReadMemory(0, hub.currentFrame().frame().memoryByteSize())))
+        .exoBytes(Optional.of(hub.romLex().getCodeByMetadata(contract)))
```

Comment:
Is `hub.romLex().getCodeByMetadata(contract)` already initialized to the `initCode` of the `CREATE` when this is executed ?

Reply:
yes, just before : 
![Uploading image.png…]()